src/components/App.tsx

```diff
@@ -400,10 +406,6 @@ let currentScrollBars: ScrollBars = { horizontal: null, vertical: null };
 let touchTimeout = 0;
 let invalidateContextMenu = false;
 
-// remove this hack when we can sync render & resizeObserver (state update)
-// to rAF. See #5439
-let THROTTLE_NEXT_RENDER = true;
```

Comment:
replacing this with `window.EXCALIDRAW_THROTTLE_NEXT_RENDER` isn't correct here. Basically it's about both the https://github.com/excalidraw/excalidraw/issues/5439, and making throttling host-app configurable. The configuration is done through a global flag currently (hence `window.EXCALIDRAW_THROTTLE_NEXT_RENDER`)